app/controllers/users.controller.js

```diff
@@ -284,3 +284,13 @@ exports.isUserBanned = function (req, res, next) {
         next();
     });
 };
+
+exports.getCurrentUser = function (req, res) {
+    var user = AuthUtils.getUserFromToken(req);
+    User.findById(user._id, function (err, user) {
```

Comment:
Before finding the user from the database, you should check and see if the var, `user` is null or undefined, and if it is, send a 401 back to the client.

`
if (!user) {
    return res.status(401).send('unauthorized');
}
`